getting-started/scaling-out.md

```diff
@@ -0,0 +1,128 @@
+
+### Information Schema for Data Nodes
+
+The data nodes that have been added to the database can be found by
+querying the `timesscaledb_information.data_node`.
+
+| Column            | Type    | Description                                               |
+|-------------------|---------|-----------------------------------------------------------|
+| `node_name`       | name    | Data node name                                            |
+| `owner`           | oid     |                                                           |
+| `options`         | text[]  | Options used when creating the data node                  |
+| `node_up`         | boolean | Data node responds to ping                                |
+| `num_dist_tables` | bigint  | Number of distributed hypertables that use this data node |
+| `num_dist_chunks` | numeric | Total number of chunks stored in data node                |
+| `total_dist_size` | text    | Total amount of data stored in data node                  |
+
+## Working with Distributed Hypertables
+
+As previously mentioned, distributed hypertables are similar to normal
+hypertable with the difference that distributed hypertables will
+partition the data and store it on remote data nodes. This means that
+to create a hypertable, you need to have added at least one data node
+before starting to create distributed hypertables.
+
+### Creating Distributed Hypertables
+
+As with normal hypertables, you have to start by creating a regular
+table to hold the data, then transform it into a distributed
+hypertable. In contrast to when creating a normal hypertable, we use
+the function
+[`create_distributed_hypertable`][create_distributed_hypertable].
+
+```sql
+CREATE TABLE conditions (
+  time        TIMESTAMPTZ       NOT NULL,
+  location    TEXT              NOT NULL,
+  temperature DOUBLE PRECISION  NULL,
+  humidity    DOUBLE PRECISION  NULL
+);
+
+SELECT create_distributed_hypertable('conditions', 'time');
+```
+
+You can now start inserting data into the distributed hypertable and
+it will automatically be partitioned on the available data nodes. You
+can find more information for how work with data in hypertables in the
+chapter [Creating Hypertables][creating-hypertables].
+
+### Changing the Number of Data Nodes for a Distributed Hypertable
+
+By default, distributed hypertables will use all available data nodes,
+but individual data nodes can be added and removed from already
+created distributed hypertables using
+[`attach_data_node`][attach_data_node] and
+[`detach_data_node`][detach_data_node].
+
+For example, if you add a new data node, it will not automatically be
+added to existing distributed hypertables, so it is necssary to attach
+it explicitly.
+
+```sql
+SELECT add_data_node('node3', host => 'dn3.example.com');
+SELECT attach_data_node('node3', hypertable => 'conditions');
+```
+
+Once the data node has been attached, any new rows added will also be
```

Comment:
`Add` is used for everything, which is, imho, too many. Will it be more natural to insert rows? I.e.,:

> any new rows inserted will